### ISSUE TITLE
Make a separate file for email templates

### ISSUE THREAD

**Issue by poornaraob** [EnCiv/undebate#100]
In order to organise documents filing  under google drive_undebate folder, team decided to make a separate file for email templates. 
Link for the google drive: https://drive.google.com/drive/folders/1w6vn6xSwHAijfRvgH9HNpjVFBb8fxlwJ

**Comment by MMitsuyasu** [comment]
Here’s a link to the new folder on the Google Drive, currently called “Email Templates”:
https://drive.google.com/drive/folders/1o12l5cvwSyAZTQ13bWBcZ9hxGmZQ8g7y?usp=sharing

I’ll work on setting up docs for each email next, with the draft copy sketched out at the Seattle “St. Hack-Tricks Day” event, if any.

**Comment by MMitsuyasu** [comment]
[Separate Google Docs](https://drive.google.com/drive/folders/1o12l5cvwSyAZTQ13bWBcZ9hxGmZQ8g7y?usp=sharing) are now established for each email we've identified thus far.

Any existing work from the Seattle St. Hack-trick's Day event has been added to each file.

I also included a "[Communication Plan](https://docs.google.com/document/d/1ZOOriUGHh8ga3dOPHm79g8NMZk4outkTl5q3uVIftyQ/edit?usp=sharing)" document in the folder as an overview and to capture any material useful to the whole communication project.

Waiting now on an update from @ddfridley about the meeting with Ballotpedia before proceeding further on communication related tasks.
